src/redux/actions/actionsCreators.js

```diff
@@ -39,3 +44,31 @@ export function loadTodos(todos) {
         })
     }
 }
+
+export const addTodoStart = () => ({
+    type: ADD_TODO_START
+})
+
+export const addTodoSuccess = (todoToBeAdded) => ({
+    type: ADD_TODO_SUCCESS,
+    payload: todoToBeAdded
+})
+
+export const addTodoError = (error) => ({
+    type: ADD_TODO_FAILURE,
+    payload: error
+})
+
+export function addTodo(todo) {
+    return dispatch => {
+        dispatch(addTodoStart());
+        postTodoToServer(todo)
+        .then(response => {
+            dispatch(addTodoSuccess(response.config.data))
```

Comment:
Just seeing that there is no `;` but other lines contains it, I would prefer to have it consistent over all files or maybe there is a specific reason to not have it here.

Reply:
Why is it `response.config.data` from you swagger definition there is no `config` key.